frontend/src/v5/ui/routes/dashboard/projects/containers/uploadFileForm/uploadList/uploadListItem/components/uploadListItemTitle/uploadListItemTitle.styles.ts

```diff
@@ -22,8 +22,12 @@ export const DashboardListItemTitle = styled(ListItemTitle)`
 	min-width: 90px;
 	padding-right: 5px;
 	box-sizing: border-box;
+
 	.MuiButton-root {
 		cursor: default;
 		text-decoration-line: none;
+		text-overflow: ellipsis;
+		overflow: hidden;
+		display: inline-block;
 	}
```

Comment:
![image](https://user-images.githubusercontent.com/62481250/211859638-aa2c43b1-ca64-48ab-9a7f-126d29c48e2e.png)
The latest revision is still not truncated (In the destination input of the upload file modal)